CruCentralCoast/Community/Ministry Teams/MinistryTeamsTVC.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  MinistryTeamsTVC.swift
+//  CruCentralCoast
+//
+//  Created by Landon Gerrits on 5/30/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+
+class MinistryTeamsTVC: UITableViewController {
```

Comment:
Delete this class since it isn't used anymore. We don't want it to enter the repo